Section 3 - Terraform Provisioners/local-exec.tf

```diff
@@ -1,8 +1,37 @@
 resource "aws_instance" "myec2" {
-   ami = "ami-082b5a644766e0e6f"
-   instance_type = "t2.micro"
+  ami           = "ami-082b5a644766e0e6f"
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure IMDSv1 is disabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances that have IMDSv1 enabled are vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
In AWS Console - 
 1. When launching a new instance in the Amazon EC2 console, select the following options on the Configure Instance Details page:
     a. Under Advanced Details, for Metadata accessible, select Enabled.
     b. For Metadata version, select V2.

source: tenablecs-